src/mem.rs

```diff
@@ -455,6 +469,16 @@ impl Decompress {
         self.inner.decompress(input, output, flush)
     }
 
+    /// Similar to [`Self::decompress`] but accepts uninitialized buffer
```

Comment:
If we are adding a new public method, I think it should also be made clear why we'd use one over the other.
Alternatively, we can make it private.

Reply:
Thanks, updated the doc to explain why they should use that